<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>RzxLoader</class>
 <widget class="QWidget" name="RzxLoader" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="3" >
    <widget class="QTreeWidget" name="treeModules" >
     <property name="iconSize" >
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights" >
      <bool>true</bool>
     </property>
     <property name="itemsExpandable" >
      <bool>false</bool>
     </property>
     <column>
      <property name="text" >
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Version</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Description</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>571</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" >
    <widget class="QPushButton" name="btnReload" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Reload</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QPushButton" name="btnLoad" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Load</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="0" />
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
